### DISCUSSION TITLE
Floating Button "Back to top"

### DISCUSSION THREAD

**Discussion by Mawobi** [SchwarzIT/onyx#1205] [Ideas]
A floating button "Back to top" would be useful.
Should appear automatically when you scroll down the page.
Could look like this:
![image](https://github.com/SchwarzIT/onyx/assets/16415986/03dec4a1-4ca3-4a70-b4b0-b5ed3570eca1)

**Comment by larsrickert** [comment]
I like the idea :)
@SchwarzIT/onyx-ux  What do you think?

**Reply by mj-hof** [reply]
Thank you for your input. Unfortunately, this type of button is not a current priority, but it will be considered sooner or later :)

**Reply by mj-hof** [reply]
@JoCa96 Please have a look if this is an a11y issue/topic

**Comment by larsrickert** [comment]
@Mawobi We concepted a floating action button which is already available in Figma that can also be used for such "Back to top" button.

You can track the technical implementation in #3565
![Bildschirmfoto 2025-06-18 um 11 53 06](https://github.com/user-attachments/assets/a125a380-090f-486b-b92f-285cb7f9877c)
